package.json

```diff
@@ -15,7 +15,7 @@
     "doc": "mmdc -i docs/*.mmd",
     "lint": "tslint -c tslint.json 'src/**/*.ts'",
     "prepush": "npm run lint && npm run build",
-    "release": "npm run prepush && np",
+    "release": "npm run prepush && npm link",
```

Comment:
Why? np is a command to publish this package to npm, related to release. I don't think we should do this here.

Reply:
Aw gotcha